README.md

```diff
@@ -29,99 +39,145 @@ When you run the tool, it will automatically notify you if there is a newer vers
 ### No installation
 
 ```bash
-npx ika
+npx asari
 ```
 
 The `npx` command lets you use this tool without installing it. However, each time you use `npx` it downloads the whole package from the npm registry, which takes a while. That's why global installation is reccommended.
 
-> Note: If this tool is globally installed, `npx ika` will use that globally installed version rather than downloading.
+> Note: If this tool is globally installed, `npx asari` will use that globally installed version rather than downloading.
 
 ## Commands
 
 ```bash
-ika
+npx asari
```

Comment:
Can't you just use `asari` without the `npx`?

Reply:
That's correct. The README advises `npx asari` because: 

> Note: If this tool is globally installed, npx asari will use that globally installed version rather than downloading.

Reply:
This way, the examples work for both globally-installed, and non-installed cases.